packages/core/docs/recipes/validation.md

```diff
@@ -0,0 +1,205 @@
+# Use of Validator plugin
+Please read [this document](../reference/service/plugins/validator.md) For additional configuration and API reference.
+
+## Recipes:
+* [Service configuration](#service-configuration)
+* [Request validation](#request-validation)
+* [Query string validation](#query-string-validation)
+* [Object validation](#object-validation)
+
+## Service configuration
+### 1. Enable plugin
+Enable plugin by adding it into `config.plugins`:
+```js
+// src/configs/plugins.js
+exports.plugins = [
+  'validator',
+];
+```
+
+### 2. Plugin configuration
+Define plugin configuration inside your configuration folder:
+```js
+// src/configs/validator.js
+exports.validator = {
+  schemas: ['../schemas'],
+}
+
+```
+All schemas from `${ProjectRoot}/schemas` are loaded into validator and available by file name or `$id` defined inside. 
+
+### 3. Define JSON Schemas
+All schemas are defined using [JSON Schema Standart](https://json-schema.org/specification.html) and all validation keywords before `draft-07`.
+
+#### `TODO` Object
```

Comment:
The content of the recipe is quite descriptive, although I am thinking about changing the structure a bit. It's not a required change though, just my point of view. 
Now there are some schemas definitions first, both for the object validation and the request validation. What I suggest is to make the iteration of understanding a single concept as short as we can: you may define the schema and provide the validation example right next to it, then the same thing with incoming request validation.
Also it would be great to explain the case, like what are we going to do here, some  short intro -- because it might be not clear why do we start with defining the `TODO` object schema. Smth like: 
- In this recipe we're going to validate create and list operations over the resource of some todos.
- Let's define the object schema
- Let's validate a todo object first right in the controller (and there goes the this.validate(...))
- Now let's validate it using request params schema (and here you make the schema for automatic validation and reference the todo object schema)
- Move on with another `list` action validation

Like, make it work -> make it good -> make it fast approach, but without the last part 😄